js/adapt-contrib-gmcq.js

```diff
@@ -49,6 +49,51 @@ define([
 
       this.$el.toggleClass('has-column-layout', isLarge);
       this.$('.js-mcq-item').css('width', isLarge ? (100 / columns) + '%' : '');
+    },
+
+    updateMarking: function () {
+
+      var isInteractive = this.model.isInteractive();
+      var canShowMarking = this.model.get('_canShowMarking');
+      var ariaLabels = Adapt.course.get('_globals')._accessibility._ariaLabels;
+
+      this.model.getChildren().each(function (itemModel) {
```

Comment:
Prefer native alias:
```suggestion
      this.model.getChildren().forEach(function(itemModel) {
```